### DISCUSSION TITLE
Datasource Variables to allow true multi-value selection

### DISCUSSION THREAD

**Discussion by JohnyChevyEG** [grafana/grafana#71494] [General]
I was curious if there was a future feature add to allow multiple data sources to be selected from a data source variable.  I have multiple Prometheus data sources that have the same metrics and I would like to include them all in the same panel without having to add a query for each data source.  If I set up a variable to search only for my Prometheus data sources but it only queries the first data source checked.  From what I have seen, this multi-value option will work when repeating to multiple panels but I want all nodes on the same panel.  Thank you.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82605. Feel free to continue the discussion around this there. Thank you!
